Plugins/StomtPlugin/Source/StomtPlugin/Private/StomtAPI.cpp

```diff
@@ -104,72 +137,93 @@ void UStomtAPI::SendStomtLabels(UStomt * stomt)
 	if (!stomt->GetServersideID().IsEmpty() && stomt->GetLabels().Max() > 0)
 	{
 		//Reset Request
-		this->request->ResetResponseData();
-		this->request->ResetRequestData();
+		this->Request->ResetResponseData();
+		this->Request->ResetRequestData();
 
 		UE_LOG(LogTemp, Warning, TEXT("nice1"));
-		this->request->SetVerb(ERequestVerb::POST);
-		this->request->SetHeader(TEXT("appid"), this->GetAppID() );
+		this->Request->SetVerb(ERequestVerb::POST);
+		this->Request->SetHeader(TEXT("appid"), this->GetAppID() );
 
-		this->request->GetRequestObject()->SetField(TEXT("name"),	UStomtJsonValue::ConstructJsonValueString(	this, TEXT("newlabeltest")	));
-		this->request->GetRequestObject()->SetField(TEXT("as_target_owner"), UStomtJsonValue::ConstructJsonValueString(this, TEXT("true")));
+		this->Request->GetRequestObject()->SetField(TEXT("name"),	UStomtJsonValue::ConstructJsonValueString(	this, TEXT("newlabeltest")	));
+		this->Request->GetRequestObject()->SetField(TEXT("as_target_owner"), UStomtJsonValue::ConstructJsonValueString(this, TEXT("true")));
 
-		this->request->ProcessURL( this->GetRestURL().Append(TEXT("/stomts/")).Append(stomt->GetServersideID()).Append(TEXT("/labels") ) );
+		this->Request->ProcessURL( this->GetRestURL().Append(TEXT("/stomts/")).Append(stomt->GetServersideID()).Append(TEXT("/labels") ) );
 	}
 	*/
 }
 
-void UStomtAPI::RequestTarget(FString targetID)
+UStomtRestRequest* UStomtAPI::RequestTarget(FString TargetID)
 {
-	this->request->SetVerb(ERequestVerb::GET);
-	this->request->SetHeader(TEXT("appid"), this->GetAppID() );
+	UStomtRestRequest* request = NewObject<UStomtRestRequest>();
```

Comment:
Move the creation of a new GET request in a own function, and attach the access token to get request too.